src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,89 @@
+.product-card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+  color: $main-accent;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 16px;
+
+  &__image-block {
+    margin: auto;
+  }
+
+  &__image {
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    margin-top: 16px;
+    width: 160px;
+    height: 134px;
```

Comment:
The margin and width properties for `&__image` should be grouped together, and the height should follow after them. Refer to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules) for the correct order of properties.